parser/parser.go

```diff
@@ -389,31 +408,28 @@ func (p *parser) VisitConstantLiteral(ctx *gen.ConstantLiteralContext) interface
 	return p.reportError(ctx, "invalid literal")
 }
 
-// Visit a parse tree produced by CELParser#exprList.
-func (p *parser) VisitExprList(ctx *gen.ExprListContext) interface{} {
```

Comment:
Just want to confirm that there is no other function call this function.

Reply:
Yep, it looks like this was dead code since it has no callers.